lms/static/sass/shared/_footer-edx.scss

```diff
@@ -7,17 +7,67 @@
 $edx-footer-link-color: $link-color;
 $edx-footer-bg-color: rgb(252,252,252);
 
-
 // Aggressively scoped for Drupal
 // ==============================
 // These styles are being loaded on Drupal, LMS and WordPress
 // sites so the scope has to be aggressive to override default
 // CMS styles
 footer#footer-edx-v3 {
   background: $edx-footer-bg-color;
-  padding: 20px;
+  padding: 15px 0;
   border-top: 1px solid $courseware-button-border-color;
 
+  .container-bootstrap {
+    padding-right: 15px;
+    padding-left: 15px;
+    margin-right: auto;
+    margin-left: auto;
+  }
+  ul li h3{
+    font-size: 16px ;
+    font-weight: 600;
+    padding:5px 0; 
+    color: #646464;
```

Comment:
Use the color variables and font sizes defined [here](https://github.com/edx/edx-platform/blob/master/common/static/sass/edx-pattern-library-shims/base/_variables.scss)

Reply:
Agreed. Don't hard-code font sizes, colors etc.